src/views/Books/BookList.js

```diff
@@ -20,9 +21,11 @@ function BookList() {
   return (
     <ul className="book-list" aria-label="book list">
       {books.map((book) => (
-        <li key={book.book_id}>
-          <Book book={book} />
-        </li>
+        <a key={book.book_id} href={`/books/${book.book_id}`}>
```

Comment:
Remember to use Link here within react-router-dom to prevent a full page reload.